api/v1alpha3/suite_test.go

```diff
@@ -29,29 +28,15 @@ import (
 )
 
 var (
-	env *envtest.Environment
+	env envtest.Environment
```

Comment:
Why is is not a pointer anymore?

Reply:
Changed it back. I tried to find a way to avoid the double pointer in the input struct, but it didn't work anyway.